x/sync/metrics.go

```diff
@@ -19,7 +19,7 @@ var (
 type SyncMetrics interface {
 	RequestFailed()
 	RequestMade()
-	RequestSucceeded()
+	RequestSucceeded() //TODO only need one of succeeded/failed
```

Comment:
Think we don't need both failed + success metric since we can infer one off the other if we know the `n` of requests + one of either the success/failed count